.github/workflows/spelling.yml

```diff
@@ -0,0 +1,31 @@
+name: Spell checking
+on:
+  pull_request_target:
+  push:
+  issue_comment:
+    types: [created]
```

Comment:
This seems like it's doing way too much? Our goal is to spell check the docs themselves when a pull request is filed.